rollup.config.js

```diff
@@ -18,13 +18,13 @@ const external = [
 ];
 
 export default {
-	exports: 'default',
```

Comment:
I think this results in a commonjs/umd bundle with exports like:

```js
exports.Provider = Provider;
exports.connect = connect;
exports.connectAdvanced = connectAdvanced;
exports.default = { Provider, connect, connectAdvanced };
```

Might be wrong though.